x-pack/plugins/monitoring/server/kibana_monitoring/bulk_uploader.js

```diff
@@ -140,6 +161,7 @@ export class BulkUploader {
   }
 
   static combineStatsLegacy(payload) {
+    throw 'This is deprecated';
```

Comment:
Anytime something gets thrown, it should be an Error:
```
throw new Error('combineStatsLegacy is deprecated');
```

Reply:
Gotcha. The `throw` here won't remain when this is merged. I should have probably prefaced the request to review this code with a warning that the code isn't in it's final form - just laying the foundations for the idea

Reply:
No problem. It's fine for checking nothing is inadvertently calling this method.